UnityAds/src/main/java/com/mopub/mobileads/UnityInterstitial.java

```diff
@@ -128,19 +129,7 @@ protected void show() {
             return;
         }
 
-        if (UnityAds.isReady(mPlacementId)) {
-            // Lets Unity Ads know when ads succeeds to show
-            MediationMetaData metadata = new MediationMetaData(mContext);
-            metadata.setOrdinal(++impressionOrdinal);
-            metadata.commit();
-
-            UnityAds.show((Activity) mContext, mPlacementId, mUnityShowListener);
-        } else {
-            // Lets Unity Ads know when ads fail to show
-            MediationMetaData metadata = new MediationMetaData(mContext);
-            metadata.setMissedImpressionOrdinal(++missedImpressionOrdinal);
-            metadata.commit();
-
+        if (mPlacementId == null) {
```

Comment:
This, as well as checking for a configured placement are handled in the Webview, and cab be routed down here from there.

Having this context through our SDK can be valuable 👍